source/daplink/drag-n-drop/vfs_user.c

```diff
@@ -286,6 +286,9 @@ static uint32_t read_file_details_txt(uint32_t sector_offset, uint8_t *data, uin
 #endif
 #ifdef HID_ENDPOINT
     pos += util_write_string(buf + pos, ", HID");
+#endif
+#if (USBD_WEBUSB_ENABLE)
```

Comment:
Should this be `#ifdef WEBUSB_HID_ENDPOINT`?

Reply:
Thanks. I fixed it in commit: https://github.com/ARMmbed/DAPLink/pull/377/commits/894248d